pkg/pcaps/common.go

```diff
@@ -71,7 +71,7 @@ func getItemIndexFromEvent(event *trace.Event, itemType PcapType) interface{} {
 		return ret
 	}
 
-	return new(interface{})
+	return ""
```

Comment:
Oh shit, I don't have a test for pcap code yet (there is an issue opened for it). Will have to test to make sure files are being kept opened while LRU entry exists (and being evicted correctly). Will need to test this tomorrow :\ just to make sure.

Reply:
Might be best I drop the LRU commit then, maybe add to a separate PR, as it's not strictly needed functionality wise here. If you want you could even add pcaps test + this in one PR, might be best.